### ISSUE TITLE
rotation_rfc3339 parameter not conflicting with other rotation_ parameters e.g rotation_days, rotation_minutes etc in time_rotating resource

### ISSUE THREAD

**Issue by ccsandhanshive** [hashicorp/terraform-provider-time#108]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v1.0.0
provider registry.terraform.io/hashicorp/time v0.7.2
### Affected Resource(s)

* time_rotating

### Terraform Configuration Files

```hcl
resource "time_rotating" "example" {
  rotation_minutes = 1
  rotation_rfc3339 = "2022-08-25T03:31:00Z"
}
```

### Debug Output
 ```
>terraform apply

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the
following symbols:
  + create

Terraform will perform the following actions:

  # time_rotating.example will be created
  + resource "time_rotating" "example" {
      + day              = (known after apply)
      + hour             = (known after apply)
      + id               = (known after apply)
      + minute           = (known after apply)
      + month            = (known after apply)
      + rfc3339          = (known after apply)
      + rotation_minutes = 1
      + rotation_rfc3339 = "2022-08-25T03:31:00Z"
      + second           = (known after apply)
      + unix             = (known after apply)
      + year             = (known after apply)
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

time_rotating.example: Creating...
time_rotating.example: Creation complete after 0s [id=2022-07-26T04:21:05Z]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

>terraform show
# time_rotating.example:
resource "time_rotating" "example" {
    day              = 26
    hour             = 4
    id               = "2022-07-26T04:21:05Z"
    minute           = 21
    month            = 7
    rfc3339          = "2022-07-26T04:21:05Z"
    rotation_minutes = 1
    rotation_rfc3339 = "2022-08-25T03:31:00Z"
    second           = 5
    unix             = 1658809265
    year             = 2022
}
 ```

### Expected Behavior

- As per terraform document `rotation_rfc3339` should conflict with other rotation_ parameters like `rotation_days`, `rotation_minutes` parameters. 
- If this is expected behavior in that case **"Conflicts with other rotation_ arguments."** statement should be removed from `rotation_rfc3339` parameter description

### Actual Behavior

- Resource is created successfully 
- As per our observation it is preferring to `rotation_rfc3339` because resource is not destroying even after waiting for minutes specified in `rotation_minutes` parameter e.g. 1 minute

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by bendbennett** [comment]
Hi @ccsandhanshive thank you for identifying that the schema attribute descriptions are incorrect. The descriptions have been amended in a recent PR (#105) and will be released in the near future.

**Comment by SBGoods** [comment]
Hi @ccsandhanshive, thank you again for bringing this issue to our attention. As @bendbennett has mentioned, this issue has been fixed in PR #105 and will be included in the next release (see milestone [v0.8.0](https://github.com/hashicorp/terraform-provider-time/milestone/4) ). Please let us know if you have any other questions.
